app/scripts/controllers/table-search-controller.js

```diff
@@ -211,6 +213,20 @@ angular.module('groongaAdminApp')
         }
       }
 
+      function toggleAllColumns() {
+        $scope.table.allColumns.forEach(function(columnInfo) {
+          columnInfo.output = $scope.toggleAllColumnsChecked;
+        });
+        if (!$scope.toggleAllColumnsChecked) {
+          $scope.table.allColumns.filter(function(columnInfo) {
```

Comment:
I'm sorry but I couldn't understand the reason why `_id` and `_key` are treated as special cases...
Could you describe more about this? (What is "most cases"?)